addon/styles/_frost-guage.scss

```diff
@@ -0,0 +1,33 @@
+.frost-guage {
+  position: relative;
+
+  &-text {
```

Comment:
I also thought that `.frost-gauge-selected` would be preferrable over `.frost-gauge.selected`, wouldn't it, since `selected` is pretty generic, and would be easy for someone to style it w/o scoping it first on accident.